### PR TITLE
Add section on how to use lntop

### PR DESCRIPTION
**Pull request by VajraOfIndra**
#### What

This PR adds a section to the lntop bonus guide that explains how to use it (what keys to use and how to navigate the displays).

### Why

The navigation is not necessarily easy to work out and the explanations are a bit hard to find! Laying out clear instructions would help users to make the most of lntop :)

#### How

Used the 'keybindings.go' file from the lntop repo ([link](https://github.com/edouardparis/lntop/blob/650f1420d35b16f97f52dc828f8c9a095bc6a7c9/ui/keybindings.go)) to work out what keys are used for what purpose.

Added a section to the guide listing all the keys and what they do.

#### Scope

- [ ] significant change to core configuration
- [x] independent bonus guide
- [ ] simple bug fix

#### Test & maintenance

Test:
* Install lntop
* Go through the explanations and try out each key and see if result is as described

Maintenance: Occasional updates when/if the lntop developper add or changes the keys

#### Animated GIF (optional)

keyboard

### CODE REVIEW

File: bonus/lightning/lntop.md [hunk 1]
```diff
@@ -91,4 +91,39 @@ $ lntop
...
 ---
 
+### lntop in action
+
+To use all the functionalities of lntop, use the following keys:
+  
+* **F1 (or h)** = Display an "About" page and a list of keyboard keys to use (press F1 again to exit this screen)
+  
+* **F2 (or m)** = Display a Menu bar on the left
+  1) Navigate the Menu with the up and down keys (see below); there are three options:
```
Comment (Stadicus) [inline bonus/lightning/lntop.md:101]: The numbered list is not shown correctly due to wrong MD syntax:

image

Two options:

* add blank lines between numbered list items: this shows okay, but the `1)` is not recognized as a numbered list, it's just text without proper indentation on small screens
* use the standard MD syntax:
  ```
  1. text
  1. text
  1. text
  ```

File: bonus/lightning/lntop.md [hunk 2]
```diff
@@ -91,4 +91,39 @@ $ lntop
...
+  
+* **F2 (or m)** = Display a Menu bar on the left
+  1) Navigate the Menu with the up and down keys (see below); there are three options:  
+    *  CHANNEL = (the home page/default view), a table of all channels
+    *  TRANSAC = a table of lightning transactions
+    *  ROUTING = a table of routing event as they happen (no historical events shown, and any displayed event will be deleted if you quit lntop)
+  2) Press Enter to see the desired view
+  3) Press F2 to enter the desired view and exit the left Menu bar
+     
+* **←, →, ↑, ↓** =
```
Comment (Stadicus) [inline bonus/lightning/lntop.md:108]: Maybe add "Arrow keys: " before the symbols to make it more clear?

File: bonus/lightning/lntop.md [hunk 3]
```diff
@@ -91,4 +91,39 @@ $ lntop
...
+    * CHANNEL = Display detailed information about a channel
+    * TRANSAC = Display detailed information about a transaction
+    * ROUTING = Display detailed information about a forwarded payment
+ 
+* **a** = Sort out column, ascending order
+ 
+* **d** = Sort out column, descending order
+
+* **F10 (or q or Ctrl+C)** = Quit lntop
+
```
Comment (Stadicus) [inline bonus/lightning/lntop.md:128]: In the main guide, I started adding a bit of whitespace after the guide and the bottom navigation:

```

<br /><br />

---

<< Back: ....
```

**Review by Stadicus — CHANGES_REQUESTED** [review]
Nice addition, I think that's very helpful!

One change is needed regarding proper MD syntax for the numbered list, and a few additional (optional) nits.

File: bonus/lightning/lntop.md (hunk 1 above)
Comment (VajraOfIndra) [reply bonus/lightning/lntop.md]: Ok, I'll use option 2 from now on, thanks.

File: bonus/lightning/lntop.md (hunk 2 above)
Comment (VajraOfIndra) [reply bonus/lightning/lntop.md]: Good idea, done

File: bonus/lightning/lntop.md (hunk 3 above)
Comment (VajraOfIndra) [reply bonus/lightning/lntop.md:128]: Ok, I'll remember this for the other guides

**Review by Stadicus — APPROVED** [review]
LGTM! :)

**Pull request merged**